app/src/main/kotlin/com/intive/picover/profile/view/ProfileScreen.kt

```diff
@@ -36,3 +69,55 @@ fun ProfileScreen(viewModel: ProfileViewModel) {
 		}
 	}
 }
+
+@Composable
+private fun LabelText(
+	@StringRes textResourceId: Int,
+) {
+	Text(
+		text = stringResource(id = textResourceId),
+		modifier = Modifier
+			.fillMaxWidth(),
+		fontSize = 20.sp,
+		fontWeight = FontWeight.Bold,
+		textAlign = TextAlign.Center,
+	)
+}
+
+@Composable
+private fun GreyDivider() {
+	Divider(
+		modifier = Modifier
+			.fillMaxWidth()
+			.padding(all = 14.dp),
+		thickness = 1.dp,
+	)
+}
+
+@Composable
+private fun ShowAvatar(
```

Comment:
I think we should avoid such naming in UI composable functions, like with show prefix, that looks to me more like a function triggering something to be shown, or intention to do something for example show some popup that is even not connected strictly with our app. Just Avatar or UserAvatar should be ok, to be more focused on the content itself, not an action.

Reply:
Done 😄